daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-overview.md

```diff
@@ -47,6 +51,47 @@ apps:
 
 For a more in-depth example and explanation of the template properties, see [Multi-app template]({{< ref multi-app-template.md >}}).
 
+{{% /codetab %}}
+
+{{% codetab %}}
+<!--kubernetes-->
+With Multi-App Run, you can start multiple applications in a Kubernetes development or test environment using a single `dapr run -f -k` command using a template file. The template file describes how to start multiple applications as if you had run many separate CLI `run`commands. By default, this template file is called `dapr.yaml`. 
+
+## Multi-App Run template file
+
+When you execute `dapr run -f .`, Dapr generates the multi-app template file (named `dapr.yaml`) in the `.dapr/deploy` directory within each application. Each time you run `dapr run -f -k .`, the generated Kubernetes Deployment YAML file is overwritten. run all the applications.
+
+You can name template file with preferred name other than the default. For example `dapr run -f -k ./<your-preferred-file-name>.yaml`.
+
+The following example includes some of the template properties you can customize for your applications. In the example, you can simultaneously launch 2 applications with app IDs of `nodeapp` and `pythonapp`.
+
+```yaml
+version: 1
+common:
+apps:
+  - appID: nodeapp
+    appDirPath: ./nodeapp/
+    appPort: 3000
+    containerImage: ghcr.io/dapr/samples/hello-k8s-node:latest
+    createService: true
+    env:
+      APP_PORT: 3000
+  - appID: pythonapp
+    appDirPath: ./pythonapp/
+    containerImage: ghcr.io/dapr/samples/hello-k8s-python:latest
+```
+
+> **Note:** 
```

Comment:
Should change this to a mark-down note format

Reply:
The note/alert shortcode doesn't work inside {{< tabs >}}